### PR TITLE
Configurable photometry buttons frontend implementation

### PR DESCRIPTION
**Pull request by jadalilleboe**
Adds a form for custom filter groups and renders buttons for those:
ezgif com-gif-maker (6)
This is only the frontend implementation, next PR will persist these filter groups in the database (perhaps in the preferences JSON column in the User table)

### CODE REVIEW

File: skyportal/plot.py [hunk 1]
```diff
@@ -747,7 +774,7 @@ def photometry_plot(obj_id, user, width=600, device="browser"):
         renderers.append(model_dict[key])
         imhover.renderers.append(model_dict[key])
 
-        key = label + 'obserr' + str(i)
+        key = label + '~obserr' + str(i)
```
Comment (guynir42) [inline skyportal/plot.py:777]: Make this an f-string.

File: skyportal/plot.py [hunk 2]
```diff
@@ -1022,7 +1050,7 @@ def photometry_plot(obj_id, user, width=600, device="browser"):
...
```
Comment (guynir42) [inline skyportal/plot.py:1053]: f-string

File: skyportal/plot.py [hunk 3]
```diff
@@ -1240,7 +1269,7 @@ def photometry_plot(obj_id, user, width=600, device="browser"):
 
             # phase plotting
             for ph in ['a', 'b']:
-                key = label + 'fold' + ph + f'{i}'
+                key = label + '~fold' + ph + f'{i}'
```
Comment (guynir42) [inline skyportal/plot.py:1272]: f-string

**Review by guynir42 — APPROVED** [review]
Look great. Nice work. Minor changes and then feel free to merge.

**Comment by jadalilleboe** [comment]
Thanks @guynir42!

**Pull request merged**